test/e2e/apiserverextraargs.go

```diff
@@ -0,0 +1,18 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"github.com/aws/eks-anywhere/test/framework"
+)
+
+func runApiServerExtraArgsUpgradeFlow(test *framework.ClusterE2ETest, clusterOpts ...[]framework.ClusterE2ETestOpt) {
+	test.GenerateClusterConfig()
+	test.CreateCluster()
+	for _, opts := range clusterOpts {
```

Comment:
Should we validate that the kcp object indeed has these flags set and not set?

Reply:
Yes, we should. Added the validation in the flow